sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -342,8 +346,8 @@ def _update_dict(self, obj: dict, fields_data: dict):
         for attr_name, attr_value in obj.items():
             attr_info = fields_data.get(attr_name, None)
             if attr_info is None:
-                raise ValueError("Provided unknown attribute `%s`" % (attr_name,))
-            if hasattr(attr_info.type_, "__attrs_attrs__"):
+                logger.warning("Provided unknown attribute `%s`", attr_name)
```

Comment:
This still is not showing warning when providing undefined config options in class constructor - this is the reason we missed the issue - should have been failed.

Reply:
this code flow is only on update, not creation.

once we get to the update flow, the unknown fields are long gone by now

Reply:
yes, that's why we missed the issue - if it was in creation flow it would fail in artifact test I suppose.
The same, now we won't be seeing warnings when existing scylla.yaml has something we don't specify.

Reply:
we will see those warning once updates are taking place